main.py

```diff
@@ -84,10 +108,88 @@
             # The other members are given access through another Mailbox.
             # You could make the work easier if you also posted a cc channel message already over here.
 
-            # TODO
+            if ' ' not in element.name:
+
+                if element.owner not in element.members:
+                    element.members.append(element.owner)
+                for buddy in element.settlers:
+                    if buddy not in element.members:
```

Comment:
Wait, why does a user need to be in _both_ element.settlers **and** element.members? surely element.settlers would suffice?

Reply:
element.settlers is the list of people for whom the channel will be their main channel (the channel in which they can use their role-related commands).
element.members is the list of people that are added to the channel.

Though all settlers should be added to the channel anyway, I'm leaving this open, may we have a program that sets up a gamelog as well. That's the channel I'm going to appoint for users like the regular innocent, to prevent optional spoilers when they don't get the error message in town_square. However, as gamelog is a channel that innocents should (obviously) NOT get access to, I'm for now simply copying all people in element.settlers to element.members, to get this function to be optional, may we create that gamelog channel, so that, in the future, we may have settlers that won't get added to their homechannel. Who knows.